easybuild/tools/config.py

```diff
@@ -109,6 +109,7 @@ def mk_full_default_path(name, prefix=DEFAULT_PREFIX):
         'only_blocks',
         'optarch',
         'parallel',
+        'rebuild',
```

Comment:
@cfenoy: fixed this (and the other remarks) in https://github.com/cfenoy/easybuild-framework/pull/2, please merge to get it included here?